packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/StakeEthInANutshellModal/InfoRow.tsx

```diff
@@ -0,0 +1,62 @@
+import { Badge, Paragraph, Row, useElevation } from '@trezor/components';
+import { Elevation, mapElevationToBorder, spacings, spacingsPx } from '@trezor/theme';
+import styled from 'styled-components';
+
+const ListItem = styled.li<{ $elevation: Elevation }>`
+    position: relative;
+    padding-left: ${spacingsPx.xl};
+    margin-bottom: ${spacingsPx.md};
+    list-style-type: none;
+
+    &::before {
+        position: absolute;
+        content: '';
+        left: 0;
+        top: 50%;
+        transform: translateY(-50%);
+        width: ${spacingsPx.md};
+        height: ${spacingsPx.md};
+        border-radius: 50%;
+        border: ${spacingsPx.xxs} solid ${mapElevationToBorder};
+    }
+
+    &:not(:last-child)::after {
+        position: absolute;
+        content: '';
+        left: calc(${spacingsPx.md} / 2 - ${spacingsPx.xxxs} / 2);
+        top: 100%;
+        width: ${spacingsPx.xxxs};
+        height: ${spacingsPx.md};
+        border-left: ${spacingsPx.xxxs} dashed ${mapElevationToBorder};
+    }
+`;
+
+interface InfoRowProps {
+    label: React.ReactNode;
+    content: {
+        isBadge?: boolean;
+        text: React.ReactNode;
+    };
+    hasVerticalLine?: boolean;
+}
+
+export const InfoRow = ({ label, content }: InfoRowProps) => {
+    const { elevation } = useElevation();
+
+    const displayContent = content.isBadge ? (
+        <Badge size="tiny">{content.text}</Badge>
+    ) : (
+        <Paragraph variant="tertiary" typographyStyle="hint">
+            {content.text}
+        </Paragraph>
+    );
+
+    return (
+        <ListItem $elevation={elevation}>
+            <Row justifyContent="space-between">
+                <Row gap={spacings.xs}>{label}</Row>
```

Comment:
Is this Row needed?

Reply:
no, you're right. Missed it after refactoring. Thanks!  https://github.com/trezor/trezor-suite/pull/14410/commits/3dbc96661afb16f226d41fa245ff8d0f7c840850